test/integ_tests/qiskit/0_Getting_Started_mocks.py

```diff
@@ -42,6 +42,20 @@ def pre_run_inject(mock_utils):
                     "deviceType": "QPU",
                     "deviceStatus": "ONLINE",
                     "providerName": "Test Provider"
+                },
+                {
+                    "deviceArn": "arn:aws:braket:us-west-2::device/qpu/arn/TestARN",
+                    "deviceName": "SV1",
+                    "deviceType": "SIMULATOR",
+                    "deviceStatus": "ONLINE",
+                    "providerName": "Test Provider"
+                },
+                {
+                    "deviceArn": "arn:aws:braket:us-west-2::device/qpu/arn/TestARN",
+                    "deviceName": "DM1",
+                    "deviceType": "SIMULATOR",
+                    "deviceStatus": "ONLINE",
+                    "providerName": "Test Provider"
```

Comment:
Trying to understand the change here, 
1. Why do we need the DM1 and SV1 to be repeated here? 
2. And region as `us-west-2` is correct?